### PR TITLE
Enabled csp with two non-ideal compromises

### PR DESCRIPTION
**Pull request by DonatJR**
see #27

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -294,8 +294,9 @@ play.filters {
...
 
     # The Content-Security-Policy header. If null, the header is not set.
-    #contentSecurityPolicy = "default-src 'self'; script-src 'self'; object-src 'none'; style-src 'self' https://fonts.googleapis.com https://fonts.gstatic.com; img-src 'self'; media-src 'none'; font-src https://fonts.googleapis.com https://fonts.gstatic.com; connect-src 'self'; child-src 'none'; form-action 'none'; frame-ancestors 'none'; report-uri 'none'"
-    contentSecurityPolicy = null // FIXME #15 enable security policy
+    # FIXME WARNING unsafe-inline SHOULD be disabled, but currently webpack loads styles inline so we need it
+    # FIXME WARNING unsafe-eval SHOULD be disabled, but our current Vue build uses it to compile templates (=> precompile them to render functions)
+    contentSecurityPolicy = "default-src 'self'; script-src 'self' 'unsafe-eval'; object-src 'none'; style-src 'self' https://fonts.googleapis.com https://fonts.gstatic.com 'unsafe-inline'; img-src 'self'; media-src 'none'; font-src https://fonts.googleapis.com https://fonts.gstatic.com; connect-src 'self' wss://localhost:9443; child-src 'none'; form-action 'self'; frame-ancestors 'none'; report-uri 'none';"
```
Comment (SailReal) [inline conf/application.conf]: Does `connect-src 'self' wss://localhost:9443` works even on our deployment server which is e.g. https://learn-duel.herokuapp.com/?

Comment (DonatJR) [reply conf/application.conf]: nope, it doesn't. should I change it to accept connections from `learn-duel.herokuapp.com` as well?

Comment (SailReal) [reply conf/application.conf]: yes pls and also `learn-duel-dev.herokuapp.com`

**Review by SailReal — APPROVED** [review]
Nice 👍 !!!

File: conf/application.conf (hunk 1 above)
Comment (DonatJR) [reply conf/application.conf]: done, the wildcard should work but we have to test it once it is on the develop branch and deployed

**Pull request merged**